crates/bevy_ecs/src/system/system_piping.rs

```diff
@@ -202,7 +202,7 @@ pub mod adapter {
     /// // Building a new schedule/app...
     /// # use bevy_ecs::schedule::SystemStage;
     /// # let mut sched = Schedule::default(); sched
-    /// #     .add_stage(CoreStage::Update, SystemStage::single_threaded())
+    /// #     .add_stage(CoreStage::Update, SystemStage::parallel())
```

Comment:
Why is everything being changed to parallel()?

Reply:
IIRC this panicking test was reverted to single_threaded because of task pool failures in CI.

Reply:
Yep, that was the temporary solution used in #4998 and the one proposed to users. This is no longer necessary.